ktor-utils/jsAndWasmShared/src/io/ktor/util/logging/LoggerJs.kt

```diff
@@ -28,5 +28,5 @@ public actual interface Logger {
     public actual fun trace(message: String, cause: Throwable)
 }
 
-public actual val Logger.isTraceEnabled: Boolean
-    get() = true
```

Comment:
No idea why this implementation didn't check the log level but always returned `true`.